var/spack/repos/builtin/packages/libnl/package.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+import sys
+
+
+class Libnl(AutotoolsPackage):
+    """libnl - Netlink Protocol Library Suite"""
+
+    homepage = "https://www.infradead.org/~tgr/libnl/"
+    url      = "https://github.com/thom311/libnl/releases/download/libnl3_3_0/libnl-3.3.0.tar.gz"
```

Comment:
So officially according to the website `3.2.25` is the [latest stable](https://www.infradead.org/~tgr/libnl/).  I actually had the same urls as you from github to start, but @adamjstewart has a strong preference for non-github tags (since they can be changed).

It's generally unclear to me what the state of this project is...

1. As packaged in Fedora:

```console
dnf info libnl3
Last metadata expiration check: 5 days, 7:42:11 ago on Sat May  6 11:25:52 2017.
Installed Packages
Name        : libnl3
Arch        : x86_64
Epoch       : 0
Version     : 3.2.27
Release     : 1.fc23
Size        : 819 k
Repo        : @System
From repo   : updates
Summary     : Convenience library for kernel netlink sockets
URL         : http://www.infradead.org/~tgr/libnl/
License     : LGPLv2
Description : This package contains a convenience library to simplify
            : using the Linux kernel's netlink sockets interface for
            : network manipulation
```

2. Who is the maintainer?  `tgraf` or `thom311`?